syntaxes/GDScript.tmLanguage.json

```diff
@@ -502,6 +535,30 @@
                 }
             ]
         },
+        "builtin_get_nodepath_shorthand_quoted": {
+            "begin": "(&|\\^|@)([\"'])",
+            "end": "([\"'])",
+            "name": "support.function.builtin.shorthand.gdscript",
+            "beginCaptures": {
+                "1": {
+                    "name": "variable.other.enummember"
+                },
+                "2": {
+                    "name": "constant.character.escape"
+                }
+            },
+            "endCaptures": {
+                "1": {
+                    "name": "constant.character.escape"
+                }
+            },
+            "patterns": [
+                {
+                    "match": "[^%^\"^']*",
+                    "name": "constant.character.escape"
+                }
+            ]
+        },
         "builtin_get_node_shorthand_quoted": {
             "begin": "(\\$)([\"'])",
```

Comment:
Godot 4 also accepts `%"..."` and `%'...'` quoted syntax.

Reply:
Good catch.